csharp_dotnetcore/09.Message-Routing/DeploymentScripts/MsbotClone/bot.recipe

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "1.0",
+  "resources": [
+    {
```

Comment:
This looks incorrect. the name does not match your .bot file. Try re-running msbot clone. and update this.